### DISCUSSION TITLE
SUBSCAN (WEBSITE)

### DISCUSSION THREAD

**Discussion by bulo4ka8993** [itering/subscan-essentials#243] [Q&A]
Hello. I am writing about the problem on the website https://www.subscan.io/. When I enter my address, I am redirected to the Subscan Polkadot website, although I should have stayed on the Subscan website with all my tokens. Plus, there is no search window on the Subscan website itself.
![1](https://github.com/subscan-explorer/subscan-essentials/assets/123836560/1dcb29cc-5f21-4cbb-a62f-82db90806e95)

I press the "find" button and it throws me here https://www.subscan.io/account/15hjAH9GXCLMkcSxeRcbdbtuQZ59pgL6rDU73FBknBprgNR8 (this is an example addres)
![2](https://github.com/subscan-explorer/subscan-essentials/assets/123836560/19a24eef-c90a-441d-b6d7-a3119df253b2)

He writes that “Account not found on the Polkadot network” but just below it says that the address contains other assets.
![3](https://github.com/subscan-explorer/subscan-essentials/assets/123836560/1f9dc9c8-f0e5-43ce-b17c-232bf5500fdb)

By clicking on the "here" button I get to the site https://www.subscan.io/account/15hjAH9GXCLMkcSxeRcbdbtuQZ59pgL6rDU73FBknBprgNR8 As you can see, there is no search window for the address. I waited 2 days, but the problem did not go away. What's happening? And when will the problem be solved?

I emailed you with the same problem.: hello@subscan.io

**Comment by carlhong** [comment]
It is fixed.
